source/features/pr-branches.tsx

```diff
@@ -94,7 +95,9 @@ function createLink(reference: RepositoryReference): HTMLSpanElement {
 }
 
 async function init(): Promise<false | void> {
-	const prLinks = select.all('.js-issue-row .js-navigation-open[data-hovercard-type="pull_request"]');
+	const prLinks = select.all('.js-issue-row .js-navigation-open[data-hovercard-type="pull_request"]')
+		// Exclude bots
+		.filter(link => !link.parentElement?.querySelector(botSelectors.join(',')));
```

Comment:
```suggestion
		.filter(link => !link.parentElement?.querySelector(botSelectors.join()));
```
Isn't `.join` by default `','`?